backend/index.js

```diff
@@ -155,6 +157,13 @@ const whereaboutsApi = whereaboutsApiFactory(
   })
 )
 
+const communityApi = communityApiFactory(
```

Comment:
is adding to health checks a separate story?

Reply:
No, but I'll add one in a new PR